src/odin/enhancedtrippath.cc

```diff
@@ -19,6 +19,7 @@ using namespace valhalla::baldr;
 
 namespace {
 constexpr float kShortRemainingDistanceThreshold = 0.402f; // Kilometers (~quarter mile)
+constexpr int kSignificantRoadClassThreshold = 2;          // Max lower road class delta
```

Comment:
so the significance is relative to the road class that the path is on. this makes intuitive sense to me though at the lower classes does it mean driveways will look significant or is there a lower bound where everything is insignificant?

Reply:
the current goal was to ignore lower class intersecting edges from current higher class path - if needed, we could try to distinguish between lower class roads.